artdaq-core-mu2e/Overlays/DTC_Types.h

```diff
@@ -117,6 +117,8 @@ enum DTC_DebugType
 	DTC_DebugType_Invalid = 5,
 };
 
+size_t WriteDMABufferSizeWords(std::ostream& output, bool includeDMAWriteSize, size_t data_size, std::streampos& pos, bool restore_pos);
```

Comment:
This feels wrong, I don't like free functions declared in namespaces. There is a Utilities class floating around somewhere, could this go in there?